topology/Stanford University/SLAC/WT2.yaml

```
@@ -70,7 +70,8 @@ Resources:
         Primary:
           ID: cf2de3eba56782b901bade70fe8bc1c18c36dd90
           Name: Wei Yang
-    Description: This site use batch fair share to manage resource usage. The maximum
+    Description:
```

```suggestion
    Description: >
```

You'll need something like the following if you want to make this multiline

---

done.